xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/displayers/DisplayerLink.vue

```diff
@@ -91,6 +97,19 @@ export default {
     }
   },
 
+  beforeMount () {
+    this.logic.loadTranslations({
+      componentName: "displayer-link",
+      prefix: "livedata.displayer.link",
+      keys: [
+        "copy",
+        "rename",
+        "rights",
+        "delete",
+      ],
+    });
```

Comment:
This doesn't seem to be used here but above in actions displayer (where it's actually not needed).